prow/jobs/governance.yaml

```diff
@@ -230,6 +230,47 @@ presubmits: # runs on PRs
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
+    - name: pre-main-kyma-crd-check-governance
+      annotations:
+        pipeline.trigger: "pr-submit"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-kyma-crd-check-governance"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-build-pr: "true"
+        preset-dind-enabled: "true"
+      run_if_changed: 'milv.config.yaml|.md$'
+      skip_report: false
+      decorate: true
+      path_alias: github.com/kyma-project/kyma
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^master$
+        - ^main$
```

Comment:
Is this job relevant only for the main branch, or would it be needed also for release branches? As the script is on target repository, you can run it against all branches I believe. Note that if you want to run it on 2.11 and 2.12 branch you have to also include this script on release branches. 

If you don't want to do it then I'd suggest leaving it only for main as of now.